src/freertos_drivers/pic32mx/Pic32mxCan.cxx

```diff
@@ -62,8 +62,7 @@ class Pic32mxCan : public Node
         , hw_(module)
         , overrunCount_(0)
     {
-        messageFifoArea_ = malloc(
-            (config_can_tx_buffer_size() + config_can_rx_buffer_size()) * 16);
+        messageFifoArea_ = malloc(2 * 16 * CAN_TX_RX_MESSAGE_SIZE_BYTES);
```

Comment:
let's keep the configuration setting the same way for the PIC32 as all the other targets.
It's ok to replace the fixed 16 constant with CAN_TX_RX_MESSAGE_SIZE_BYTES but we should keep the config_() to allow the OVERRIDE_CONST(can_tx_buffer_size, nn); to function.